public/app/features/provisioning/OnboardingPage.tsx

```diff
@@ -0,0 +1,43 @@
+import SVG from 'react-inlinesvg';
+
+import { EmptyState, LinkButton, Alert, Stack, Text } from '@grafana/ui';
+import { Page } from 'app/core/components/Page/Page';
+
+import { SetupWarnings } from './SetupWarnings';
+import { MIGRATE_URL } from './constants';
+
+export default function OnboardingPage() {
+  return (
+    <Page
+      navId="provisioning"
+      pageNav={{ text: 'Setup provisioning', subTitle: 'Configure this instance to use provisioning' }}
+    >
+      <Page.Contents>
+        <SetupWarnings />
+        <Alert severity="info" title="Setting up this connection could cause a temporary outage">
+          When you connect your whole instance, depending on its size, the setup might make the dashboard unavailable to
+          users for up to 30 minutes. We recommend warning your users before starting the process. You can use the
+          announcement banner or your preferred channels.
```

Comment:
Can we link to the page for banners? (though I guess we'll have to add a TODO until it's moved to main?)

Reply:
If/when banners are OSS in main, we can totally revisit this -- right now there is nothing to link to